### PR TITLE
feat: new admin page for user allocations

### PR DESCRIPTION
**Pull request by bedreski**
Solves: #389 
This feature is focused in the user allocations tab, inside user details page. 

image

### CODE REVIEW

File: app/views/new_admin/users/_allocations.html.erb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        <td class="py-3">
+          <div class="flex items-center text-sm">
+            <p class="text-sm text-gray-600 dark:text-gray-400 font-semibold">
+              <%= allocation.ongoing %>
+            </p>
+          </div>
+        </td>
+        <td class="py-3">
+          <div class="flex items-center text-sm">
+            <%= link_to t('allocation.access_allocation'), '#', class: "inline-flex items-center w-full text-sm font-semibold transition-colors duration-150 underline hover:no-underline dark:hover:text-gray-200" %>
```
Comment (samkevin1) [inline app/views/new_admin/users/_allocations.html.erb:78]: the page that is supposed to be redirected to already exists. This is the path: `new_admin_user_allocation_path`

File: app/views/new_admin/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<% content_for :page_title do %>
+  <%= @user.first_and_last_name %>
+<% end %>
+<div>
+  <button class="px-4 py-2 text-sm font-medium leading-5 text-white transition-colors duration-150 border border-transparent rounded-t-lg
+  bg-primary-600 active:bg-primary-600 hover:bg-primary-700 focus:outline-none focus:ring" data-tab-id="user_details">Usuário</button>
+  <button class="px-4 py-2 text-sm font-medium leading-5 text-white transition-colors duration-150 border border-transparent rounded-t-lg 
+  bg-primary-400 active:bg-primary-600 hover:bg-primary-700 focus:outline-none focus:ring" data-tab-id="user_allocations">Alocações</button>
```
Comment (samkevin1) [inline app/views/new_admin/users/show.html.erb:8]: use I18n to translate "Alocações"

**Comment by nolleto** [comment]
Can you add some printscreen?

**Comment by bedreski** [comment]
> Can you add some printscreen?

File: spec/features/new_admin/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      visit "/new_admin/users/#{user.id}"
+      click_button('Alocações')
+      expect(page).to have_content('ALOCAÇÃO ATUAL')
+      expect(page).to have_content(allocation.project_name)
+    end 
+
+    it 'shows table of allocations', js: true do 
+      visit "/new_admin/users/#{user.id}"
+      click_button('Alocações')
+      page.has_table?("allocations_table")
```
Comment (adbatista) [inline spec/features/new_admin/users_spec.rb:54]: you are checking here but you are not doing any expectation

File: spec/features/new_admin/users_spec.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        end_at: 1.month.before,
+        user: user, 
+        ongoing: false
+      )
+    }
+
+    it 'shows allocations details with user current allocation', js: true do 
+      visit "/new_admin/users/#{user.id}"
+      click_button('Alocações')
+      expect(page).to have_content('ALOCAÇÃO ATUAL')
```
Comment (adbatista) [inline spec/features/new_admin/users_spec.rb:47]: you can use composable matchers here, this way will return a better and complete error message
```rb
 expect(page).to have_content('ALOCAÇÃO ATUAL') && have_content(allocation.project_name)
```

File: spec/features/new_admin/users_spec.rb [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+    it 'shows table of allocations', js: true do 
+      visit "/new_admin/users/#{user.id}"
+      click_button('Alocações')
+      page.has_table?("allocations_table")
+      expect(page).to have_content(another_allocation.project_name)
+      expect(page).to have_content("#{l(another_allocation.start_at)}")
+      expect(page).to have_content("#{l(another_allocation.end_at)}")
+      expect(page).to have_content("#{I18n.t(another_allocation.ongoing)}")
+      expect(page).to have_content('Acessar Alocação')
```
Comment (adbatista) [inline spec/features/new_admin/users_spec.rb:59]: same here about composable matchers

File: spec/features/new_admin/users_spec.rb (hunk 3 above)
Comment (bedreski) [reply spec/features/new_admin/users_spec.rb:54]: All good, that's fixed now :+1:

**Review by adbatista — APPROVED** [review]
Just solve conflicts and should be ok

**Pull request merged**